bin/configure-app-database.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
```

Comment:
@lorenyu  There is a lot of duplicate code with the create and configure scripts. Could try and clean this up in this PR or create another ticket.

Reply:
Yeah I agree. It's not trivial and I'm not sure how to address that right now so I'll create a ticket for it. https://github.com/navapbc/template-infra/issues/301